cnd/migrations/2019-11-11-034058_create-message-tables/up.sql

```diff
@@ -0,0 +1,91 @@
+-- Your SQL goes here
+
+CREATE TABLE rfc003_bitcoin_ethereum_bitcoin_ether_request_messages
+(
+    id INTEGER               NOT NULL PRIMARY KEY,
+    swap_id UNIQUE           NOT NULL,
+    bitcoin_network          NOT NULL,
+    ethereum_chain_id        NOT NULL,
+    bitcoin_amount           NOT NULL,
+    ether_amount             NOT NULL,
+    hash_function            NOT NULL,
+    bitcoin_refund_identity  NOT NULL,
```

Comment:
❓I'm a bit rusty in SQL, so please help me out here: 
What's the type of these fields? Does it default to the type `text`? 
Should we mention it nevertheless to avoid ambiguity?

Reply:
In SQlite, the types of a field is tracked together with the field, not the table: https://www.sqlite.org/datatype3.html

Reply:
Sorry for such a short initial reply. The reasoning of not specifying the column types (which is something that one can do in standard SQL) was that SQLite actually ignores them because it automatically determines the type and stores them accordingly with the field.

I didn't know this neither before writing this code but it actually became handy because we don't have to specify all those types but only make sure the format we store is correct :)